test/fixtures/optimizations.yml

```diff
@@ -1,3 +1,13 @@
 optim_ackley2d:
+  id: 2
   kind: SEGOMOE
-  config: <%= { :xlimits => [[-32.768, 32.768], [-32.768, 32.768]] }.to_json.inspect  %>
\ No newline at end of file
+  config: <%= { :xlimits => [[-32.768, 32.768], [-32.768, 32.768]] }.to_json.inspect  %>
+
+optim_unexpected_id:
+  id: -2.28
```

Comment:
Indeed there is no such thing as an `id` being equal to `-2.28`! Not sure why you need that.

Reply:
I tried to find every test i could think of, but if the ids are completely secure, no need to check for unexpected id, I agree.